java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/protocol/Server.java

```diff
@@ -452,6 +451,11 @@ public void showStatusBarMessage(ShowStatusMessageParams params) {
             return CompletableFuture.completedFuture(params.getValue());
         }
 
+        @Override
+        public void notifyTestProgress(TestProgressParams params) {
+            logWarning(params);
+        }
+
         @Override
         public NbCodeClientCapabilities getNbCodeCapabilities() {
```

Comment:
Shouldn't the additional messages in the LSP protocol be reflected in the capabilities? CCing @sdedic ...

Reply:
Done. Thanks for pointing this out.